pkg/contracts/src/CVStrategy.sol

```diff
@@ -315,11 +316,13 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
 
     function deactivatePoints() public {
         _deactivatePoints(msg.sender);
+        emit PointsDeactivated(msg.sender);
     }
 
     function deactivatePoints(address _member) external {
         onlyRegistryCommunity();
         _deactivatePoints(_member);
+        emit PointsDeactivated(_member);
     }
 
     function _deactivatePoints(address _member) internal {
```

Comment:
`PointsDeactivated` can be emitted just once in the `_deactivatePoints` instead in both funcs.